bot/exts/evergreen/timed.py

```diff
@@ -0,0 +1,36 @@
+from copy import copy
+from time import perf_counter
+
+from discord import Message
+from discord.ext import commands
+
+
+class TimedCommands(commands.Cog):
+    """Time the command execution of a command."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    @staticmethod
+    async def create_execution_context(ctx: commands.Context, command: str) -> commands.Context:
+        """Get a new execution context for a command."""
+        msg: Message = copy(ctx.message)
+        msg._update({"content": f"{ctx.prefix}{command}"})
```

Comment:
```suggestion
        msg.content = f"{ctx.prefix}{command}"

Reply:
discord.py doesn't appreciate when we directly set attributes of things like messages, which is why i'm using the update function made specifically for that

Reply:
I don't like that we're using a private method here, plus, the only thing `_update` does is set `message.content = value`. So this is the same thing here.